scripts/tests/test_windows_native_runtime_deps.py

```
@@ -0,0 +1,121 @@
+                    "platform": {"os": "windows", "arch": "x86_64"},
+                    "backend": {"kind": "vulkan"},
+                    "libraries": [
+                        "lib/ggml-vulkan.dll",
+                        "lib/libstdc++-6.dll",
+                        "lib/llama.dll",
+                    ],
+                }
+            }
+            (artifact / "manifest.json").write_text(json.dumps(manifest), encoding="utf-8")
+
+            result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)